# AOC-day-1
Advent of code day 1 simple solution in python

I used exception handling to solve it as while typecasting the number into int read by input file which are in string, error comes for \n.
On getting exception the numbers are summed and stored in an array and the loop continues again after clearing the temporary array named a.
